pkgs/by-name/ca/castero/package.nix

```diff
@@ -39,7 +39,11 @@ python3.pkgs.buildPythonApplication rec {
     "tests"
   ];
 
-  disabledTests = [ "test_datafile_download" ];
+  disabledTests = [
+    "test_datafile_download"
+    "test_display_get_input_str"
+    "test_display_y_n"
```

Comment:
```suggestion
    "test_display_y_n"
    "test_display_get_y_n"
```

Reply:
This was a typo, fixed in the latest commit. Sorry for any delays.

Reply:
Thank you so much for your time!